src/cms/views/dashboard/admin_dashboard_view.py

```diff
@@ -34,6 +37,18 @@ def get(self, request, *args, **kwargs):
         :return: The rendered template response
         :rtype: ~django.template.response.TemplateResponse
         """
+        all_feedback = Feedback.objects.filter(is_technical=True)[:5]
+        admin_feedback_exists = Feedback.objects.filter(is_technical=True).exists
 
         val = "To be defined"
```

Comment:
This should also be resolved/finalized before merging. Not sure where this should lead to?

Reply:
This `val` was introduced in 2018 (d88edb3f2c6a2492560d135dfbec75fef3b3b1e4) and ignored since then^^
I think this is not necessarily part of this PR, but if you want to, you can just delete this row.